tests/unit/test_charm.py

```diff
@@ -202,10 +200,37 @@ def test_given_service_template_when_install_then_srsenb_service_file_is_rendere
             "Restart=always\n"
             "RestartSec=1\n"
             "User=root\n"
-            "ExecStart=/build/srsenb/src/srsenb --enb.name=dummyENB01 --enb.mcc=901 --enb.mnc=70 --enb_files.rr_config=/config/rr.conf --enb_files.sib_config=/config/sib.conf --enb_files.drb_config=/config/drb.conf /config/enb.conf --rf.device_name=zmq --rf.device_args=fail_on_disconnect=true,tx_port=tcp://*:2000,rx_port=tcp://localhost:2001,id=enb,base_srate=23.04e6\n\n"  # noqa: E501, W505
+            "ExecStart=/build/srsenb/src/srsenb --enb.gtp_bind_addr=10.0.0.8 --enb.s1c_bind_addr=10.0.0.8 --enb.name=dummyENB01 --enb.mcc=001 --enb.mnc=01 --enb_files.rr_config=/config/rr.conf --enb_files.sib_config=/config/sib.conf --enb_files.drb_config=/config/drb.conf /config/enb.conf --rf.device_name=zmq --rf.device_args=fail_on_disconnect=true,tx_port=tcp://*:2000,rx_port=tcp://localhost:2001,id=enb,base_srate=23.04e6\n\n"  # noqa: E501, W505
             "[Install]\n"
             "WantedBy=multi-user.target"
         )
+        self.assertEqual(mock_open_write_srsenb_service.written_data, srsenb_expected_service)
+
+    @patch("charm.get_iface_ip_address")
+    @patch("charm.service_active", side_effect=[True, False, True, True])
+    @patch("os.mkdir")
+    @patch("shutil.copy")
+    @patch("shutil.rmtree")
+    @patch("subprocess.run")
+    def test_given_service_template_when_attach_ue_action_emitted_then_srsue_service_file_is_rendered(  # noqa: E501
+        self, _, __, ___, ____, _____, patch_get_iface_ip_address
+    ):
+        with open("templates/srsue.service", "r") as f:
+            srsue_service_content = f.read()
+
+        with patch("builtins.open") as mock_open:
+            mock_open_read_srsue_template = MockOpen(read_data=srsue_service_content)
+            mock_open_write_srsue_service = MockOpen()
+            mock_open.side_effect = [
+                mock_open_read_srsue_template,
+                mock_open_write_srsue_service,
+            ]
+            event = Mock()
+            dummy_ue_ipv4_address = "192.168.128.13"
```

Comment:
This line and the next aren't needed I believe

Reply:
Fixed